data/data_loader.py

```diff
@@ -1,25 +1,26 @@
 import numpy as np
```

Comment:
High-level comment: better practice to use `data` (with specific naming convention for which data splits are being used) and `labels` in place of X and Y.